spec/21-http_response_header_format.md

```diff
@@ -47,47 +46,37 @@ The following `version-format` definition is used for version `00`.
 version-format   = trace-id "-" child-id "-" trace-flags
 trace-id         = 32HEXDIGLC  ; 16 bytes array identifier. All zeroes forbidden
 child-id         = 16HEXDIGLC  ; 8 bytes array identifier. All zeroes forbidden
-trace-flags      = 2HEXDIGLC   ; 8 bit flags. Currently, only one bit is used. See below for details
+trace-flags      = 2HEXDIGLC   ; 8 bit flags. See below for details
 ```
 
 #### trace-id
 
-This is the ID of the whole trace forest and is used to uniquely identify a <a href="#dfn-distributed-traces">distributed trace</a> through a system. It is represented as a 16-byte array, for example, `4bf92f3577b34da6a3ce929d0e0e4736`. All bytes as zero (`00000000000000000000000000000000`) is considered an invalid value.
-
-If the `trace-id` value is invalid (for example if it contains non-allowed characters or all zeros), tracing systems MUST ignore the `traceresponse`.
+The format and requirements for this are the same as those of the trace-id field in the `traceparent` request header.
 
-See [considerations for trace-id field generation](#considerations-for-trace-id-field-generation) for recommendations on how to operate with `trace-id`.
+For details, see the trace-id section under [traceparent Header Field Values](#traceparent-Header-Field-Values).
 
 #### child-id
 
-This is the ID of the operation of the callee (in some systems known as the span id) and is used to uniquely identify an operation within a trace. It is represented as an 8-byte array, for example, `00f067aa0ba902b7`. All bytes as zero (`0000000000000000`) is considered an invalid value.
+This is the ID of the operation of the callee (in some tracing systems, this is known as the `span-id`, where a `span` is the execution of a client request) and is used to uniquely identify an operation within a trace. It is represented as an 8-byte array, for example, `00f067aa0ba902b7`. All bytes as zero (`0000000000000000`) is considered an invalid value.
+
+Vendors MUST ignore the `traceresponse` header when the `child-id` is invalid (for example, if it contains non-lowercase hex characters).
 
 #### trace-flags
 
-An <a data-cite='!BIT-FIELD#firstHeading'>8-bit field</a> that provides additional information about how the callee handled the trace such as sampling, trace level, etc. These flags are recommendations given by the callee rather than strict rules to follow for three reasons:
+Similar to the `trace-flags` field in the `traceparent` request header, this field is hex-encoded.
```

Comment:
_suggestion:_

Maybe we can combine the two comments on the format (hex encode, 8-bit) here, instead of splitting them across two paragraphs with the list of available flags in the middle? 

We could add the 8-bit aspect here and shorten the paragraph below accordingly. 

```suggestion
Similar to the `trace-flags` field in the `traceparent` request header, this is a hex-encoded <a data-cite='!BIT-FIELD#firstHeading'>8-bit field</a> that provides information about how a callee handled the trace. The same requirement to properly mask the bit field value when interpreting it applies here as well.
```

The bit 

> The same requirement to properly mask the bit-field value when interpreting it applies here as well.

is probably debatable, as we have this spelled out in trace-parent/trace-flags already. I personally feel it is worth mentioning it again.